source/assets/css/foundation/_site.scss

```diff
@@ -14,26 +24,47 @@
 html {
   @extend %text;
   overflow-x: hidden;
+
   text: {
     rendering: optimizeLegibility;
     align: center;
   }
-  background: image-url("textures/grey-prism.svg") center;
+
+  background: image-url("textures/grey-prism.svg");
   -webkit-tap-highlight-color: rgba($black, 0);
 
-  &.toolkit-baseline { @include debug-vertical-alignment; }
-}
+  &.toolkit-baseline {
+    @include debug-vertical-alignment;
+  }
 
-body { background: none; }
+}
 
-@mixin theme-selection { background: $color-background-shade; }
+body {
+  background: none;
+}
 
-::-moz-selection { @include theme-selection; }
-::selection      { @include theme-selection; }
+@mixin theme-selection {
+  background: $color-background-shade;
+}
 
+::-moz-selection {
+  @include theme-selection;
+}
 
+::selection {
+  @include theme-selection;
+}
 
 @include breakpoint($tablet-large) {
+
   html,
```

Comment:
I went ahead and merged your PR, but just noticed that you changed the selectors from being on their own line — prefer them to stay as is. Will change it back later. :)